### PR TITLE
added a customer model

### PR DESCRIPTION
**Pull request by Evans-Munatsa**
- Created a customer model for individual customers

### CODE REVIEW

**Comment by ryannealmes** [comment]
This is not good enough. I want first_name and last_name and I want to see validation on your attributes as well.

**Comment by ryannealmes** [comment]
You have added a customer spec, but you have not tests. Please checkout https://github.com/thoughtbot/shoulda-matchers for testing models.

**Comment by ryannealmes** [comment]
Further there is no CRUD functionality for customers via the front end. Is this something that is still coming?

**Comment by ryannealmes** [comment]
The 1 thing I would like to say I am happy about is the PR is small and relates to only 1 thing. This is great at it really speeds up reviewing your work. Please sort out the comments above, rebase and push the changes.

**Comment by Evans-Munatsa** [comment]
The CRUD functionality for the customer is going to follow in a different branch

File: spec/models/customer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe Customer, type: :model do
+
+  let(:customer) { FactoryGirl.build(:customer) }
+
+  describe "validates customer info" do
```
Comment (ryannealmes) [inline spec/models/customer_spec.rb]: This is much much better, but see if you can figure out how to use shoulda matchers to validate presence.

**Comment by ryannealmes** [comment]
Much better @Evans-Munatsa. Only 1 comment. See if you can figure it out.

File: app/models/customer.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Customer < ActiveRecord::Base
+  validates :first_name, presence: true
```
Comment (ryannealmes) [inline app/models/customer.rb]: See if you can figure out how to refactor this into a single line.

File: app/models/customer.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class Customer < ActiveRecord::Base
+  validates :first_name, :last_name, :phone_number, presence: true
```
Comment (ryannealmes) [inline app/models/customer.rb]: This is better. Checkout http://apidock.com/rails/ActiveRecord/Validations/ClassMethods/validates_presence_of

File: spec/models/customer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe Customer, type: :model do
+
+  let(:customer) { FactoryGirl.build(:customer) }
+
+  describe "responds to the field" do
+    it { expect(customer).to respond_to('first_name') }
```
Comment (ryannealmes) [inline spec/models/customer_spec.rb]: This is good, but you are not checking the validation.

File: spec/models/customer_spec.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe Customer, type: :model do
+
+  let(:customer) { FactoryGirl.build(:customer) }
+
+  describe "responds to the field" do
+    it { expect(customer).to validate_presence_of(:first_name) }
```
Comment (ryannealmes) [inline spec/models/customer_spec.rb]: You shouldn't have included the `responds` as well. Please update.

**Comment by ryannealmes** [comment]
Please rebase.

File: spec/models/customer_spec.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe Customer, type: :model do
+
+  let(:customer) { FactoryGirl.build(:customer) }
+
+  describe "validate presence of infomation to the field" do
+    it { expect(customer).to validate_presence_of(:first_name) }
```
Comment (ryannealmes) [inline spec/models/customer_spec.rb]: You still haven't updated this.

**Pull request merged**